api/v1alpha1/virtualmachinepublishrequest_types.go

```diff
@@ -278,6 +278,14 @@ type VirtualMachinePublishRequestStatus struct {
 	Conditions []Condition `json:"conditions,omitempty"`
 }
 
+func (vmpr *VirtualMachinePublishRequest) GetConditions() Conditions {
```

Comment:
This is overkil, and I'm not sure why it's required.

Reply:
Why do we use a getter? Is it for interface mocks? If not, then refactor it to just grab the field.

Reply:
Sorry I thought I replied it before. :( We use `Getter` is to be able to get conditions for all vmop resources. (https://github.com/vmware-tanzu/vm-operator/blob/main/pkg/conditions/getter.go#L38) This is to keep consistent with other resources, such as [VMI](https://github.com/vmware-tanzu/vm-operator/blob/main/api/v1alpha1/virtualmachineimage_types.go#L147) and [VM](https://github.com/vmware-tanzu/vm-operator/blob/main/api/v1alpha1/virtualmachine_types.go#L471). But yeah, this is overkill. Refactored it.